rosdep/python.yaml

```diff
@@ -6008,6 +6008,16 @@ python3-bitarray:
   gentoo: [dev-python/bitarray]
   nixos: [python3Packages.bitarray]
   ubuntu: [python3-bitarray]
+python3-bitstring:
+  debian:
+    '*': [python3-bitstring]
+  gentoo: [dev-python/bitstring]
+  nixos: [python39Packages.bitstring]
```

Comment:
My understanding is that almost all of our `nixos` keys use `python3Packages.something.`  So I think this should be:

```suggestion
  nixos: [python3Packages.bitstring]
```

(altenatively you could just remove this; NixOS isn't required)

Reply:
I am still learning NixOS and was expecting `python3Packages.X`. Searching I only finding this
![shutter_019](https://user-images.githubusercontent.com/413937/181797078-f155d466-47e0-4d2d-8617-8af3444cf37d.png)

Could I be misinterpreting the search results? Is the `python3Packages,bitstring` mapped to the 3.9 or 3.10 version somehow? I'll have access to a NixOS box soon and can test to see if `python3Packages,bitstring` does anything. :)

Reply:
I don't know either. Have a feeling that it maybe packaged wrong :thinking: 

I'll remove it from this PR.